### PR TITLE
results of T-compiler planning meeting today.

### PR DESCRIPTION
**Pull request by pnkfelix**

### CODE REVIEW

**Review by wesleywiser — APPROVED** [review]
lgtm

File: posts/inside-rust/2022-09-23-compiler-team-sep-oct-steering-cycle.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+[compiler-team#544]: https://github.com/rust-lang/compiler-team/issues/544
+[compiler-team#540]: https://github.com/rust-lang/compiler-team/issues/540
+
+### Details
+
+Every fourth Friday, the Rust compiler team decides how
+it is going to use its scheduled steering and design meeting time over the next
+three Fridays.
+
+On Friday, 10 June, we will do a quarterly [review of the open P-high issues][compiler-team#559].
```
Comment (ehuss) [inline posts/inside-rust/2022-09-23-compiler-team-sep-oct-steering-cycle.md:40]: I'm guessing this was intended to be "On Friday, 30 September"?

Comment (pnkfelix) [reply posts/inside-rust/2022-09-23-compiler-team-sep-oct-steering-cycle.md:40]: 🤦

Comment (pnkfelix) [reply posts/inside-rust/2022-09-23-compiler-team-sep-oct-steering-cycle.md:40]: https://github.com/rust-lang/blog.rust-lang.org/pull/1037

**Pull request merged**